docs/tutorials/build-a-dapp/prepare.md

```diff
@@ -40,9 +40,11 @@ comparing it with the hash stored on chain.
 
 ![File Hash](assets/tutorials/build-a-dapp/file-hash.png)
 
-To add to this, blockchains also provide a robust identity system. So when a file digest is stored
-on the blockchain, we can also record which user uploaded that digest. This allows that user to
-later prove that they were the original person to claim the file.
+To add to this, blockchains also provide a robust identity systems though accounts that map to 
+[public keys](https://en.wikipedia.org/wiki/Public-key_cryptography), and associations built on
+top of those keys (see the identity pallet](https://github.com/paritytech/substrate/tree/v3.0.0/frame/identity).
```

Comment:
Fix link: [see the identity pallet](https://github.com/paritytech/substrate/tree/v3.0.0/frame/identity)